src/components/Reports/Reports.js

```diff
@@ -3,6 +3,7 @@ import ReportsFilters from './ReportsFilters/ReportsFilters';
 import ReportsBox from './ReportsBox/ReportsBox';
 import ReportsPageRanking from './ReportsPageRanking/ReportsPageRanking';
 import ReportsTrafficSources from './ReportsTrafficSources/ReportsTrafficSources';
+// import ReportsDailyVisits from './ReportsDailyVisits/ReportsDailyVisits';
```

Comment:
Could you add a comment about the reason why it is commented?

Reply:
fixed